worker/codalabworker/main.py

```diff
@@ -70,9 +70,8 @@ def main():
     parser.add_argument(
         '--max-dependencies-serialized-length',
         type=int,
-        default=60000,
-        help='Maximum length of serialized json of dependency list of worker '
-        '(e.g., 50, 30000, 60000).',
+        default=10000000,
```

Comment:
How come this wasn't an issue before?

Reply:
potential reasons:
- old worker stored less state about dependencies, allowing more dependencies to be serialized in this size
- it did fire up a lot but because the system was more threaded constantly cleaning up dependencies didn't trip it up.

I don't really know at this point, and I'm not even convinced this is the reason we're having issues but looking at logs I saw this getting triggered too frequently and there's no reason to limit this to something like 58K if the actual dependencies themselves aren't taking too much space. I'm not even convinced we need this at all but didn't want to risk removing it.